src/frontend/src/lib/components/icons/OisyWalletLogo.svelte

```diff
@@ -1,1033 +1,51 @@
-			fill="url(#paint57_linear_2135_430)"
-		/>
-		<path
-			d="M40.5206 10.125L40.5242 10.1226L40.063 9.43229L40.0593 9.43472L40.5206 10.125Z"
-			fill="url(#paint58_linear_2135_430)"
-		/>
-		<path
-			d="M36.2002 7.21222L28.7449 14.6675C28.9486 14.855 29.1445 15.0508 29.332 15.2545L36.9599 7.62665C36.7107 7.48214 36.4574 7.34393 36.2002 7.21222Z"
-			fill="url(#paint59_linear_2135_430)"
-		/>
-		<path
-			d="M33.1753 6.02261L27.8759 13.9537C27.6528 13.7905 27.4227 13.6364 27.186 13.4918L32.3219 5.80544C32.6093 5.87045 32.8938 5.94291 33.1753 6.02261Z"
-			fill="url(#paint60_linear_2135_430)"
-		/>
-		<path
-			d="M34.5676 3.93887L34.568 3.93829L33.8777 3.47706L33.8773 3.47764L34.5676 3.93887Z"
-			fill="url(#paint61_linear_2135_430)"
-		/>
-		<path
-			d="M30.8032 1.83458L30.8036 1.83349L30.0366 1.51579L30.0362 1.51688L30.8032 1.83458Z"
-			fill="url(#paint62_linear_2135_430)"
-		/>
-		<path
-			d="M29.3222 5.40997L26.1945 12.9608C25.9444 12.8446 25.6886 12.7384 25.4277 12.6428L28.4287 5.39765C28.4998 5.39675 28.571 5.3963 28.6423 5.3963C28.8701 5.3963 29.0967 5.40089 29.3222 5.40997Z"
-			fill="url(#paint63_linear_2135_430)"
-		/>
-		<path
-			d="M25.6744 5.66081L24.3505 12.3165C24.0831 12.2518 23.8115 12.1979 23.5361 12.1553L24.7913 5.8452C25.0827 5.77598 25.3772 5.71444 25.6744 5.66081Z"
-			fill="url(#paint64_linear_2135_430)"
-		/>
-		<path
-			d="M26.7002 0.503663L26.7 0.504534L25.8858 0.342573L25.886 0.341702L26.7002 0.503663Z"
-			fill="url(#paint65_linear_2135_430)"
-		/>
-		<path
-			d="M119.723 12.8834L125.071 19.4418L130.432 12.8834H138.895L128.636 24.9254V31.1709H121.506V24.9254L111.246 12.8834H119.723Z"
-			fill="white"
-		/>
-		<path
-			d="M96.6048 18.884C96.6048 19.6868 98.8227 19.7957 101.694 19.9318C106.796 20.1903 113.967 20.5441 113.953 25.1296C113.953 30.3546 107.966 31.5928 101.544 31.5928C95.1352 31.5792 89.815 30.8036 89.1347 25.1296H96.6048C97.4212 26.5447 99.3125 26.912 101.544 26.912C103.762 26.912 106.483 26.5447 106.483 25.1296C106.483 24.3268 104.265 24.2043 101.394 24.0682C96.2918 23.8097 89.1211 23.4559 89.1347 18.884C89.1347 13.659 95.1216 12.4072 101.544 12.4072C107.953 12.448 113.273 13.1692 113.953 18.884H106.483C105.667 17.4417 103.776 17.1016 101.544 17.1016C99.3261 17.1016 96.6048 17.4281 96.6048 18.884Z"
-			fill="white"
-		/>
-		<path d="M80.6306 12.8563H87.7606V31.1438H80.6306V12.8563Z" fill="white" />
-		<path
-			d="M66.5741 12.4072C73.4592 12.4072 79.0515 14.421 79.0515 21.3333V22.6667C79.0787 29.579 73.4455 31.5792 66.5741 31.5928C59.6891 31.5792 54.0559 29.579 54.0831 22.6667V21.3333C54.0559 14.4074 59.6891 12.4208 66.5741 12.4072ZM71.9216 22.2177V21.7687C71.9624 18.7888 69.486 18.2445 66.5741 18.2037C63.6487 18.2445 61.1722 18.7888 61.2131 21.7687V22.2177C61.1722 25.2112 63.6487 25.7555 66.5741 25.7963C69.486 25.7555 71.9624 25.2112 71.9216 22.2177Z"
-			fill="white"
-		/>
-		<path
-			fill-rule="evenodd"
+<script lang="ts">
+	export let hideName = false;
```

Comment:
suggestion: rename to `hideTagline` maybe? We don't really hide the name but rather the "selling phrase".

Reply:
well, we are hiding both:

1) when logged out: we hide only the "selling phrase"
2) when logged in: we hide the "selling phrase" and the Oisy name; only the logo is left

maybe invert the varibale and call it `onlyLogo` or something similar?